tests/tt_eager/python_api_testing/unit_testing/fallback_ops/test_argmax_min_op.py

```diff
@@ -0,0 +1,52 @@
+# SPDX-FileCopyrightText: © 2023 Tenstorrent Inc.
+
+# SPDX-License-Identifier: Apache-2.0
+
+import torch
+import tt_lib as ttl
+from tests.tt_eager.python_api_testing.sweep_tests import (
+    comparison_funcs,
+)
+from loguru import logger
+import pytest
+import tt_lib.fallback_ops
+
+
+@pytest.mark.parametrize("on_device", [True, False])
+@pytest.mark.parametrize(
+    "input_shapes",
+    (
+        (torch.Size([1, 1, 32, 32])),
+        (torch.Size([1, 1, 320, 384])),
+        (torch.Size([1, 3, 320, 384])),
+    ),
+)
+@pytest.mark.parametrize("arg_kind", ["max", "min"])
+class TestArgMaxMinOps:
+    @pytest.mark.parametrize("dim", [-2, -1, 0, 1, 2, 3])
+    @pytest.mark.parametrize("keepdim", [True, False])
+    def test_argmax_min_fallback(self, input_shapes, dim, keepdim, arg_kind, on_device, device):
+        torch.manual_seed(1234)
+
+        x = torch.randint(low=0, high=100, size=input_shapes)
+        # Test on host RM
+        t0 = ttl.tensor.Tensor(
+            x,
+            ttl.tensor.DataType.UINT32,
+        )
+        if on_device:
+            t0 = t0.to(device)
+        if arg_kind == "max":
+            t1 = ttl.fallback_ops.torch_argmax(t0, dim, keepdim)
+            pt_out = torch.argmax(x, dim, keepdim)
+        elif arg_kind == "min":
+            t1 = ttl.fallback_ops.torch_argmin(t0, dim, keepdim)
+            pt_out = torch.argmin(x, dim, keepdim)
+        if keepdim == False:
+            pt_out = pt_out.unsqueeze(-1)
+
+        output = t1.cpu().to(ttl.tensor.Layout.ROW_MAJOR).to_torch()
+        comp_pass, _ = comparison_funcs.comp_equal(pt_out, output)
+        _, comp_out = comparison_funcs.comp_allclose_and_pcc(pt_out, output)
```

Comment:
comp_equal returns the allclose/pcc output as well so there isn't a need to run these two separately